src/components/LeftBar/LeftBar.styles.ts

```diff
@@ -1,17 +1,14 @@
 import { styled } from '@mui/system';
-import { Drawer, ListItem, Box } from '@mui/material';
+import { ListItem, Box } from '@mui/material';
 
 import theme from 'styles/theme';
 import { variables } from 'styles/variables';
 
-import { LEFT_BAR_WIDTH } from './LeftBar.const';
-
-export const StyledDrawer = styled(Drawer)`
-  .drawer-paper {
-    background-color: ${variables.palette.surface1};
-    width: ${LEFT_BAR_WIDTH};
```

Comment:
Why did you remove variable? It helps to define width of the navigation panel in one place and if it changed, also changed left margin for the other content.